services/dy-static-file-server/.dockerignore

```diff
@@ -1,3 +1,5 @@
 *
 !src/
 !docker/
+!static-content/
+!requirements_app.txt
```

Comment:
we usually have a ```requirements``` folder.
Then you could actually set
```
!**/requirements/
```
and anything in requirements folder would go in. Better than having a ```requirements_app.txt``` and then another ```requirements_whatever.txt```

Reply:
I can move them to a requirements folder, but I will still only put one file in.